<?xml version="1.0"?>
<launch>
  <node pkg="usb_cam" type="usb_cam_node" name="camera" output="screen">
    <param name="camera_name" value="elecom_ucam"/>
    <param name="camera_frame_id" value="camera_link"/>
    <param name="video_device" value="/dev/video0"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
    <param name="pixel_format" value="yuyv"/>
    <param name="io_method" value="mmap"/>
  </node>
  <node pkg="tf" type="static_transform_publisher" name="camera_transform_publisher" args="0 0 -0.478 0 0 0 1 /world /camera_link 1"/>
  <!--<node pkg="rsj_2017_block_finder" type="block_finder" name="block_finder" args="$(arg method) $(arg visual)" output="screen"/>-->
  <node pkg="rsj_2017_block_finder" type="block_finder_headless" name="block_finder" output="screen">
    <remap from="/usb_cam_node/camera_info" to="/camera/camera_info"/>
    <remap from="/usb_cam_node/image_raw" to="/camera/image_raw"/>
  </node>
</launch>
